contracts/todomanager.hpp

```diff
@@ -0,0 +1,51 @@
+#include <eosio/eosio.hpp>
+#include <eosio/system.hpp>
+#include <string>
+
+namespace todomanager
+{
+using namespace eosio;
+using std::string;
+
+class[[eosio::contract("todomanager")]] todomanager : public contract
+{
+public:
+    using contract::contract;
+
+    todomanager(name receiver, name code, datastream<const char *> ds) : contract(receiver, code, ds),
+                                                                         todos_table(receiver, code.value)
+    {
+    }
+
+    enum e_status_option : uint8_t {
+        INITIAL = 0,
+        TODO,
+        INPROGRESS,
+        DONE
+    };
+
+    typedef uint8_t status_option;
+
+    struct [[eosio::table]] todo
+    {
+        uint64_t id;
+        string description;
+        status_option status;
+
+        uint64_t primary_key() const { return id; }
```

Comment:
Add a secondary key to search by status as we will have a board and there we should show all todos from a status group

Reply:
added secondary key by_status()